wemake_python_styleguide/visitors/ast/subscripts.py

```diff
@@ -181,34 +182,44 @@ class ConsecutiveSlicesVisitor(base.BaseNodeVisitor):
     def __init__(self, *args, **kwargs) -> None:
         """Create necessary variables."""
         super().__init__(*args, **kwargs)
-        self._max_cons_slices: int = 0
-        self._first_oc: ast.Subscript
+        self._consecutives: List[ast.Subscript] = []
 
     def visit_Subscript(self, node: ast.Subscript) -> None:
         """
         Visits subscript.
 
         Raises:
-            ForbidConsecutiveSlicesViolation
+            ConsecutiveSlicesViolation
 
         """
         self._check_consecutive(node)
         self.generic_visit(node)
 
+    def _get_consecutives(self, node: ast.Subscript) -> None:
+        """Retrieves a list of all consecutive slices."""
+        ch = node.value
+
+        while isinstance(ch, ast.Subscript) and isinstance(ch.slice, ast.Slice):
+            self._consecutives.append(ch)
```

Comment:
Can we return a `list` here instead of making it a `self` prop?

Reply:
Sure, I can create a local variable that will hold the consecutive node found during the loop and use the `extend` function to save it into `self._consecutives`

Reply:
Also, it is important to keep a history of the consecutive nodes, because they are not differentiated when they are visited, and it will create multiple violations.